### ISSUE TITLE
textbox can't delete  the last word 

### ISSUE THREAD

**Issue by Wildlifes** [fabricjs/fabric.js#5592]
## Version
2.7.0

## Information about environment
 browser

## Steps to reproduce
```
new fabric.Textbox('some text', {
        width:100,
        height:30,
        splitByGrapheme: true
 })
```
## Expected Behavior
As a matter of fact, textbox was deleted along with the last text.

## Actual Behavior
![image](https://user-images.githubusercontent.com/26505011/54879303-486f4200-4e72-11e9-87ce-b5f33d2ca3cb.png)
![image](https://user-images.githubusercontent.com/26505011/54879304-4d33f600-4e72-11e9-856e-0c4b360d9316.png)
Textbox cannot delete the last word

I looked at the latest code and found that if textbox deleted the last word, the text would not disappear, the actual text had been deleted, but textbox still displayed the last word.

**Comment by asturur** [comment]
this bug is bad and needs to be fixed before of 3.0 release

**Comment by asturur** [comment]
ok fixed!
